dask_expr/tests/test_collection.py

```diff
@@ -297,6 +297,16 @@ def test_conditionals(func, pdf, df):
     assert_eq(func(pdf), func(df), check_names=False)
 
 
+@pytest.mark.parametrize("axis", ("index", 0, "columns", 1, None))
+@pytest.mark.parametrize("periods", (1, 2, None))
+def test_diff(pdf, df, axis, periods):
+    kwargs = {k: v for k, v in (("periods", periods), ("axis", axis)) if v}
+
+    actual = df.diff(**kwargs)
+    expected = pdf.diff(**kwargs)
+    assert_eq(expected, actual)
```

Comment:
Can you add a test that tests the projection?

Reply:
https://github.com/dask-contrib/dask-expr/pull/482/commits/727ee1ba63dadf47c424938f6fb314ffb558896a